src/stache/includes/components/tabs-sitelevel.html

```diff
@@ -0,0 +1,19 @@
+<p>
+  If you use tabs or vertical tabs as page navigation, each one should have an entry in the browser history with a distinct title. This enables users to select the browser's back button to return to the previous view. Tabs and vertical tabs that are used as site-level navigation contain all of a page's content and can't be dismissed.
+</p>
+
+<div class="screenshot screenshot-info">
+  <img src="~/assets/img/guidelines/back-button/tabs-backbutton.png" alt="Tabs as page navigation" style="width: 100%" />
+</div>
+
+<div class="sky-subsection-heading" style="margin-top: 20px;">Focus</div>
+
+<p>
+  Based on guidance from the <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel">W3C's WAI-ARIA Authoring Practices for tabs</a>, maintain focus on activated tabs.
+</p>
+
+<div class="sky-subsection-heading">Screen readers</div>
+
+<p>
+  Titles are announced as described in <a stacheRouterLink="." fragment="new-page-behavior">New page behavior</a>.
```

Comment:
What is this link supposed to be pointing to? I don't think we can use a link like this in an includes item because not every page is going to have a New page behavior section. The tabs topic doesn't, so this link just point to the top of the page in that topic.